enhancements/cluster-logging/logs-observability-openshift-io-apis.md

```diff
@@ -0,0 +1,602 @@
+---
+title: logs-observability-openshift-io-apis
+authors:
+- "@jcantril"
+reviewers:
+- "@alanconway, Red Hat Logging Architect"
+- "@xperimental"
+- "@syedriko"
+- "@cahartma"
+approvers:
+- "@alanconway"
+api-approvers:
+- "@alanconway"
```

Comment:
Within the core OpenShift projects, we typically use this api-approver to mean that the API review team has reviewed the enhancement and given feedback on it. Have you considered running this by the API review team?

One reason for doing so would be that the team review many designs across many different areas of focus, so can advise on patterns that others have used and advise on keeping the UX consistent across OCP. The intention of the API review is to ensure quality of API design and look for design problems that might cause problems with future upgrades or future feature additions.

While I realise this is an add-on related EP, it may still be beneficial to gather feedback from the API review team.

You can find us in #forum-api-review on slack if you have questions.

Reply:
@JoelSpeed I added this PR to slack channel but you are whomever are welcome to provide input